src/Controller/Admin/PageImportController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neusta\Pimcore\ImportExportBundle\Controller\Admin;
+
+use Neusta\Pimcore\ImportExportBundle\Documents\Import\PageImporter;
+use Neusta\Pimcore\ImportExportBundle\Toolbox\Repository\PageRepository;
+use Pimcore\Model\Document\Page;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
+
+final class PageImportController
+{
+    public function __construct(
+        private PageImporter $pageImporter,
+        private PageRepository $pageRepository,
+    ) {
+    }
+
+    #[Route(
+        '/admin/neusta/import-export/page/import',
+        name: 'neusta_pimcore_import_export_page_import',
+        methods: ['POST']
+    )]
+    public function import(Request $request): JsonResponse
+    {
+        $file = $request->files->get('file');
+        if (!$file) {
+            return new JsonResponse(['success' => false, 'message' => 'No file uploaded'], 400);
+        }
+
+        $overwrite = filter_var($request->request->get('overwrite', false), FILTER_VALIDATE_BOOLEAN);
+
+        try {
+            $yamlContent = file_get_contents($file->getPathname());
+
+            $page = $this->pageImporter->parseYaml($yamlContent);
```

Comment:
`$file` should already be an instance of `UploadedFile` right?

Then the `file_get_contents` shouldn't be necessary:

```suggestion
            $page = $this->pageImporter->parseYaml($file->getContent());
```